src/md_misc_module.f90

```diff
@@ -69,15 +69,15 @@ module md_misc
   !!    (introduced atom_vels in July, 2020.)
   !!   2022/10/04 17:27 dave
   !!    Reworking to set initial KE of ions correctly
```

Comment:
It may be okay but do you need to delete the following line ?
!! SOURCE

Reply:
Reverted